File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    
    var nop = s.replaceAll("p","").replaceAll("P","");
    var noy = s.replaceAll("y","").replaceAll("Y","");
    
    var answer = true;
    if(nop.length != noy.length) answer = false;

    return answer;
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        
        String nop = s.replace("p", "").replace("P", "");
        String noy = s.replace("y", "").replace("Y", "");
        
        boolean answer = true;

        if(nop.length() != noy.length()) answer = false;

        return answer;
    }
}